lib/IDE/CodeCompletion.cpp

```diff
@@ -2468,9 +2468,7 @@ class CompletionLookup final : public swift::VisibleDeclConsumer {
 
   void addVarDeclRef(const VarDecl *VD, DeclVisibilityKind Reason,
                      DynamicLookupInfo dynamicLookupInfo) {
-    if (!VD->hasName() ||
-        !VD->isAccessibleFrom(CurrDeclContext) ||
-        VD->shouldHideFromEditor())
```

Comment:
`isAccessibleFrom` is handled by the `AccessFilteringDeclConsumer` (added in a couple other places). `shouldHideFromEditor` is already called in `foundDecl`.